### ISSUE TITLE
some questions

### ISSUE THREAD

**Issue by 2298qq** [AAleka/ReTree#4]
Hello, I used your ReTree dataset to train a model for generating label maps. After training for over 30 epochs, the results were not ideal. However, your pre-trained model performs exceptionally well. I would like to know some details about your training process. Thank you for your outstanding work.

**Comment by AAleka** [comment]
@zxkzx25 As far as I remember, it took a lot more than 30 epochs for the model to converge (like around 100 I guess on Nvidia RTX 2080 Super). Additionally, I generated images of sizes 64x64 and 128x128 but it may go higher. However, the training time and computational cost grow as we increase the resolution, so I would recommend to start with the 64x64 images. The training parameters remain as is.
